crates/router/src/services/authentication.rs

```diff
@@ -189,6 +190,7 @@ pub struct SinglePurposeToken {
     pub origin: domain::Origin,
     pub path: Vec<TokenPurpose>,
     pub exp: u64,
+    pub tenant_id: Option<String>,
```

Comment:
Are we ensuring length this is below 64 anywhere?

Reply:
For now we are just ensuing it from DB, if the length is more than 64 charater  we will get error while insertion (signing up, creating tenant_admin etc) 
`value too long for type character varying(64)`